src/code/z_play.c

```diff
@@ -15,6 +15,14 @@ u64 D_801614D0[0xA00];
 
 void Play_SpawnScene(PlayState* this, s32 sceneNum, s32 spawn);
 
+// If R_ENABLE_PLAY_LOGS is enabled, the number "1" with a file and line number will be printed during
+// various phases of the Play update cycle. This can be helpful for debugging at what stage a problem may be occuring.
+// Due to the high number of logs occuring rapidly, the game will run slow with this enabled.
+#define PLAY_LOG(line)                        \
+    if (1 && R_ENABLE_PLAY_LOGS) {            \
+        LOG_NUM("1", 1, "../z_play.c", line); \
+    }
```

Comment:
I suspect that the 1 here is a debugging define, the same thing happens in other files. (MM's z_play has some really horrible functions to match because of these.)

Reply:
Do you have any evidence that MM's z_play is affected by this? I don't see any of these functions decompiled in MM so I haven't been able to compare them myself

Reply:
Hmm, I thought we had a file in MM but maybe not. z_std_dma does this too, though.

Reply:
This many do while (0)s is probably going to mess up the block-counting, too.

Reply:
It should be fine since we would be replacing the `if (1)` with `do while (0)`

Reply:
do while 0 does indeed work to replace if 1 in these, and is more likely what it originally was imo

Retail will tell us for sure of course